pycm/pycm_param.py

```diff
@@ -115,6 +115,8 @@
 
 CLASSES_TYPE_WARNING = "The classes is neither a list nor None so it'll be ignored."
 
+CURVE_NONE_WARNING = "The curve is undefined at some points."
```

Comment:
I suggest to make this warning clearer. For example:
"The curve axes contain non-numerical value(s)."

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/467/commits/270f77c9af42a0cd65ca272bc002c38e374244bd